environments/icds-cas/postgresql.yml

```diff
@@ -41,7 +41,6 @@ postgres_override:
   postgresql_slow_log_threshold: 1000
   postgresql_hba_entries:
     - contype: host
-      databases: replication
```

Comment:
pglogical requires superuser permissions on both replication database and the database that's being replicated from

Reply:
fun fact. `all` databases doesn't imply `replicaiton` databases  https://github.com/dimagi/commcare-cloud/pull/3080/commits/8203a0ec765fe5293f35cd2f6d2441542b77a5f9